eng/scripts/Set-VcpkgWriteModeCache.ps1

```diff
@@ -0,0 +1,17 @@
+param(
+    [string] $StorageAccountKey
+)
+Install-Module "Az.Storage" -AllowClobber -Force
+
+$ctx = New-AzStorageContext `
+    -StorageAccountName 'cppvcpkgcache' `
+    -StorageAccountKey $StorageAccountKey
+$token = New-AzStorageAccountSASToken `
```

Comment:
Do we know how long the default token time is for? We should make sure we don't want to shorten it.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/3480